spec/validators/fee/fee_type_rule_spec.rb

```diff
@@ -0,0 +1,31 @@
+RSpec.describe Fee::FeeTypeRules, type: :validator do
```

Comment:
Should this class be fee_type_rules_spec.rb (as the class it tests is fee_type_rules)?

Reply:
👍 absolutely